docs/usage/docker.md

```diff
@@ -386,7 +421,7 @@ If you have dependencies on Google Container Registry (and Artifact Registry) yo
       }
       ```
 
-##### Using short-lived access tokens
+##### Using short-lived access token / Gitlab CI / Google Cloud
```

Comment:
This section feels like a wall of text. Can you add some headings to break it up into smaller parts?

Reply:
out of scope for this pr

Reply:
I'll make a follow up PR later then.